app/api/suwon-scrape/start/route.ts

```
@@ -4,9 +4,25 @@
+import { ProfessorService } from '@/lib/supabase/services/professor-service';
+import { StudentCourseService } from '@/lib/supabase/services/student-course-service';
+import { DepartmentService } from '@/lib/supabase/services/department-service';
 
 export async function POST(req: Request) {
+
+  function parseSemesterString(semStr: string): { year: number; semester: number } {
+    const parts = semStr.split("-");
+    if (parts.length !== 2) {
+      return { year: 2023, semester: 10 };
+    }
+    const year = parseInt(parts[0], 10) || 2023;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)